### DISCUSSION TITLE
Run Gene Deletion in Human GEM

### DISCUSSION THREAD

**Discussion by Sirui-Liufu-oldone** [SysBioChalmers/Human-GEM#439] [Q&A] [answered]
I wonder how to run run the single gene deletion in Human-GEM. I have tried to run the code SingleGeneDeletion and SingleRxnDeletion as CobraToolbox offers(ENSG00000118058,ENSG00000167972, and MAR09897, respectively ). However, in the 'has effect' column, the result displayed as 0(no effect), and the flux solution also had no change compared to the original model.  I do not think the result is  reasonable for nothing changed. Is there a better way to analyze the Gene Deletion Prediction in Human GEM? Thanks for your time with this question.

**Comment by haowang-bioinfo** [comment]
Human-GEM is a generic metabolic model that collectively assemblies known human metabolism into a huge network, which serves a knowledgebase and framework for omics integration. 

For knockout and other FBA analysis, it would make more biological sense to use HumanGEM-derived context-specific models of tissues or cell lines.

**Reply by Sirui-Liufu-oldone** [reply]
Oh, I got this point!  I treated all the human cells are the same in metabolic model mistakenly.

**Comment by JonathanRob** [comment, marked as answer]
Hi @Anorve-Breaks, just to better understand your question; you listed two genes (ENSG00000118058, ENSG00000167972) and a reaction (MAR09897) - is your question targeted at these genes and reaction specifically, or on the model as a whole?

**Reply by Sirui-Liufu-oldone** [reply]
My target is at the flux changes in the whole human-GEM model while those genes and reaction are deleted in simulation. Some genetic diseases occur because of gene deletion. My original idea was to simulate the procedure of deleting those genes and associated reaction by running singleGeneDeletion and singleRxnDeletion to check whether the flux(patients' metabolites) changes throughout the whole human metabolic mode, so I recognized human-GEM as single model cell in human and all the cells play the same role in metabolism at the beginning. Then I was confused by the result and the feasibility of  applying the method for Microbe GEMs in human.  Thanks for @JonathanRob  and @haowang-bioinfo explained to that. So if I want those analysis more meaningful, I'd better to focus on specific human cell model, and do FBA analysis just like what we do in yeast /E.coli?

**Reply by JonathanRob** [reply]
Yes, generally for simulations (e.g., FBA) it is more meaningful to first contextualize/extract a cell type- or tissue-specific model from the full Human-GEM. So if there is a specific organ or tissue known to be primarily affected by the deletion of interest, then that would be the model type to generate.

That being said, the model can only simulate the direct metabolic effects of deleting a gene, with the assumption that the deletion of a gene will inactivate its associated reaction (and no isozymes can compensate). It is often the case that there is another reaction(s) through which the flux can be diverted to bypass the deleted reaction, thus resulting in no apparent effect of deletion. So even if a gene is suspected or known to have an effect when suppressed or deleted in vitro/vivo, it might not show any effect when deleted in the model.

**Comment by feiranl** [comment]
Hi @haowang-bioinfo @JonathanRob   do we need to adjust the media setup before running the single gene deletion simulation?

**Reply by JonathanRob** [reply]
@feiranl this is a bit what @haowang-bioinfo was referring to - it depends entirely on the question being asked. Since Human-GEM, unlike microbe GEMs (e.g. Yeast-GEM), is multicellular, the full (generic) Human-GEM model is not really representative of any real cell, but instead is a combination of all the metabolic capabilities of all human cells. Therefore, it is often more meaningful when running simulations or gene deletion analyses to contextualize (using e.g. tINIT) the model to represent a specific cell type (e.g., hepatocyte) - as well as set the media constraints - to reflect the experimental system of interest.

One can still run analyses such as single gene deletion on the full Human-GEM, but this would yield fewer genes with an effect, and may be interpreted as deletions that are likely to affect _any_ human cell. But even with this approach, one would still need to define media constraints - otherwise there are many ways in which the model will "cheat" by simply importing metabolites from the media. In our Human1 paper, for example, we used a relatively generic Ham's media, since it is defined and quite minimal (by mammalian cell media standards).

**Reply by feiranl** [reply]
I agree with both of you that it is more meaningful to do so in tissue level. I was just thinking about the issue: in this generic model, if we don't do any media constraint, then many metabolites can be imported which could also cause the issue here that no gene has an effect.

**Reply by JonathanRob** [reply]
Yes, exactly. With no media constraints, there will be fewer genes identified as "essential" because their function can be replaced by importing/exporting metabolites.

Also, this question depends on the objective function - biomass production is not necessarily the best or only objective that should be considered when doing such an analysis/simulation.
